src/bin/vip-config-software-update.js

```diff
@@ -54,7 +54,7 @@ const cmd = command( {
 	wildcardCommand: true,
 	usage,
 } ).examples( examples );
-cmd.option( 'yes', 'Auto-confirm update' );
+cmd.option( 'yes', 'Skip the confirmation prompt and automatically submit "y".' );
```

Comment:
```suggestion
cmd.option( 'yes', 'Skip the update confirmation prompt and automatically submit "y".' );
```
Feels more explicit and clear for me. Feel free to disregard it.

Reply:
Because the user will be looking at the `-h` menu for the command `vip config software update`, it seems redundant to me to add "update" to the description for the `--yes` option. Would it be acceptable to commit my version of the description as-is and we can add the clarifier later if we think it's needed?

Reply:
Sounds good!